## 03fully_expand_screenencodings.dpatch by  <hesso@pool.math.tu-berlin.de>
##
## DP: SCREENENCODINGS contains ${prefix} from $datadir, which needs
## DP: an extra expansion step.

@DPATCH@
diff -Naur screen-4.0.3.orig/configure.in screen-4.0.3/configure.in
--- screen-4.0.3.orig/configure.in	2008-06-01 23:40:01.000000000 +0200
+++ screen-4.0.3/configure.in	2008-06-01 23:40:03.000000000 +0200
@@ -1287,7 +1287,7 @@
 SCREENENCODINGS="\"/usr/local/lib/screen/utf8encodings\""
 if test -n "$datadir"; then
 eval SCREENENCODINGS="$datadir/screen/utf8encodings"
-SCREENENCODINGS="\"$SCREENENCODINGS\""
+eval SCREENENCODINGS="\\\"$SCREENENCODINGS\\\""
 fi
 AC_MSG_CHECKING(for the utf8-encodings location)
 AC_DEFINE_UNQUOTED(SCREENENCODINGS,$SCREENENCODINGS)
